### PR TITLE
Added info on using video directive

### LINKED ISSUE #388: Add info on using video directive to the contributor guide
document how to use the video directive created by #370 
should replace the information on using the raw html

### PR DESCRIPTION
**Pull request by ankita240796**
closes #388 

## What is included in this PR?
Added information on using video directive in contributors guide.

### CODE REVIEW

File: contributing.rst [hunk 1]
```diff
@@ -1576,15 +1576,46 @@ Video files should be put in the :file:`/vid/` directory in the source, and they
 
 The length of the videos must be less than a minute.
 
-There is no ``video`` directive to add a video, so to add a video in a document, you can do the following:
+We have a custom ``video`` directive to add a video.
```
Comment (adammichaelwood) [inline contributing.rst]: There are custom rst directives and roles for writing about rst roles and directives.

http://www.sphinx-doc.org/en/stable/domains.html#the-restructuredtext-domain

(I've used them other places in the contrib guide, so you can see how they normally appear.)

File: contributing.rst [hunk 2]
```diff
@@ -1576,15 +1576,46 @@ Video files should be put in the :file:`/vid/` directory in the source, and they
...
+
```
Comment (adammichaelwood) [inline contributing.rst]: Put a simple example (no changes to default behavior) here, before the customization details.

File: contributing.rst (hunk 1 above)
Comment (adammichaelwood) [reply contributing.rst]:
```

.. rst:directive:: .. video:: path/to/video
 
  Explanation

  ..rst:role:: autoplay

    Explanation of autoplay


```

**Comment by ankita240796** [comment]
@adammichaelwood I made the required changes. I also used the rst roles in the csv table directive.

**Pull request merged**